EIPS/eip-5218.md

```diff
@@ -0,0 +1,235 @@
+---
+eip: 5218
```

Comment:
this pr has number 5222

Reply:
I assigned 5218 because 5222 seems a bit too "nice" a number, and since you cannot prove/disprove number squatting, I assigned an unused number.

Reply:
@TimDaub Shall I resolve this conversation?

Reply:
> I assigned 5218 because 5222 seems a bit too "nice" a number, and since you cannot prove/disprove number squatting, I assigned an unused number.

Usually the burden of proof is on the accuser, why is it different here?

Reply:
EIP numbers are assigned by the editors.  It is intentionally left up to the editors to decide what number to give so that people cannot game the system.  While we do have a default rule for number selection that we use internally to simplify our lives, we often override that rule in order to defend against number sniping because in the past number sniping has lead to people spamming the repository with garbage issues in order to farm for certain numbers.

There is no "proof" necessary for anything here, the editor assigned a number is all that has happened here.

Reply:
The editor assigned a number because the initial one was "too nice" according to their own words, please see comments above. There is zero indication that the authors of this EIP spammed to end up with the number they initially did. So how exactly is the editor preventing spam here?

Reply:
I'll resolve this.
```suggestion
eip: 5222
```

Reply:
Editors assign numbers, not authors and not the PR number (even though that is what we *sometimes* use internally to simplify assignment).  There is nothing wrong with 5218 as an EIP number, and I see no reason to change it to 5222 in this case.

Reply:
It isn't possible to prove someone is gaming the system because it is a very fuzzy line, and we have a history of problems where people spam create issues on this repository to bump the number so they can snipe (which creates work for the editors who are woefully understaffed).  It is far easier to just have the simple rule of "editor picks the number" (which is what we have).

The editors try to give their reasoning for not following our usual **internal** number selection mechanism to facilitate understanding between and potentially start a conversation among editors.  Sometimes disagreements occur between editors and they are usually discussed in our fortnightly meetings.

What is pertinent about the above is that editors choose the EIP number, not authors, and there is no mechanism currently for authors to choose their preferred EIP numbers.  We have an internal mechanism for number selection that we usually follow because it is easy, but we don't stick strictly to it specifically so we can address any potential gaming of the system.

Since numbers are just a UID, we try to be over-cautious since false positives have no real cost.  This frees us from having to waste too much time thinking about system gaming as we can just freely assign numbers as we deem necessary, even if it turns out it isn't necessary for some reason.

----

It is worth noting that editors are all volunteers and no one is getting paid for their work.  We are woefully understaffed and overworked so most of our rules are around reducing editor workload as much as possible while maintaining as much fairness as we can.  I personally prioritize helping people understand the EIP process whenever I can, but it is one of the major editor time sinks.  Arguing with the editors over their decisions is more likely to cause the rules to become even more draconian, not less, as we will need to adjust them to be even stricter if enough people cause editors to spend time explaining why we reserve the right to pick numbers or external links are not allowed.

Anyone is welcome to participate in the EIP improvement process and/or EIPIP meetings where we discuss these things, but be forewarned that any solution that increases editor time or decreases fairness (even transitively) is unlikely to see much editor support.